nipype/interfaces/base.py

```diff
@@ -738,9 +729,12 @@ def _get_filecopy_info(self):
 
 
 class BaseInterfaceInputSpec(TraitedSpec):
-    ignore_exception = traits.Bool(False, desc="Print an error message instead \
-of throwing an exception in case the interface fails to run", usedefault=True,
-                                   nohash=True)
+    ignore_exception = traits.Bool(False, usedefault=True, nohash=True,
+                                   desc='Print an error message instead of throwing an exception '
+                                        'in case the interface fails to run')
+    resource_monitor = traits.Bool(True, usedefault=True, nohash=True,
+                                   desc='Disable the resource monitor for this interface '
```

Comment:
is this really saying disable when set to `True`?

also should `resource_monitor` be an input or a class attribute like `_redirect_x`?

Reply:
Yes, that `desc` is wrong.

Your question is pretty good. Why `ignore_exception` is an input? shouldn't it be a class attribute? But sure, we can make it a class attribute.

Reply:
i agree - several things have bothered me about being inputs/class attributes. 

`terminal_output`, `ignore_exception` are particular offenders. but for those we would need to go through a deprecation cycle. for this, i think we can start correctly :)

Reply:
BTW, I didn't comment on this, but back in #1371 I already played around these making them traits. And I wrote it was a good idea.

We may consider having a traited `interface.settings` object (or similar).